docs/cloud-tutorials/application-management.md

```diff
@@ -39,6 +39,10 @@ When you deploy an application, DBOS Cloud first runs [`npx dbos migrate`](../ap
 It then starts your application.
 After your application is deployed, DBOS Cloud hosts it at this URL, which is also printed by the deploy command:
 
+:::info
+Your application must not consume more than 1GB of disk space.
```

Comment:
Just curious - why is this a requirement? Wouldn't this detract some customers from using DBOS?

Reply:
And isn't this requirement repeated below?

Reply:
Just our current configuration. Can evolve in the future.

Reply:
Your application is allowed to consume "disk space"?!  What even is that?

Reply:
"must not use more .." updated